removeJobSchedules/README.md

```diff
@@ -0,0 +1,42 @@
+# Cleaning up job schedules
+This can be used for both aaS and on prem installations of CPD. This example will remove the schedule in all jobs in the specified project
```

Comment:
Nit: Typo

```suggestion
# Cleaning up job schedules
This can be used for both SaaS and on-prem installations of CPD. This example will remove the schedule in all jobs in the specified projects.
```

Reply:
Fixed in both of the readme files